app/cells/folio/console/index/actions_cell.rb

```diff
@@ -35,6 +35,11 @@ def default_actions
         icon: :edit_box,
         url: -> (record) { through_aware_console_url_for(record, action: :edit, safe: true) },
       },
+      clone: {
```

Comment:
podle nejake tridni metody bych to zaroven pridal pro patricne tridy do `default_actions`